setup.py

```diff
@@ -70,10 +70,15 @@ def run(self):
         # ... setuptools is an enigma monkey patched on a mystery
         if not os.path.exists(dest_path):
             os.makedirs(dest_path, exist_ok=True)
+
+        release_path = os.path.join(source_path, "Release")
+        if os.path.exists(release_path):
+            source_path = release_path
         self.copy_tree(source_path, dest_path)
 
+
 setup(
-    name='Rtree',
+    name='Rtree-linux',
```

Comment:
I don't understand renaming the library here.

Reply:
Oh, I was working on my fork to release the 3.9 wheels for Windows & Linux! I didn't realise that branch was linked to this PR :) I'll revert this. I've recently asked someone to help me look into the Mac wheels which would let us finish this PR, so fingers crossed